src/app/queries/mocks/hosts.mock.ts

```diff
@@ -1,41 +1,19 @@
-import { IHost, IHostsByProvider } from '../types/providers.types';
+import { IHost } from '../types/providers.types';
 
-export let MOCK_HOSTS_BY_PROVIDER: IHostsByProvider = {};
+export let MOCK_HOSTS_BY_PROVIDER: IHost[] = [];
 
 // TODO put this condition back when we don't directly import mocks into components anymore
-// if (process.env.NODE_ENV === 'test' || process.env.DATA_SOURCE === 'mock') {
-const host1: IHost = {
-  metadata: {
-    name: 'host1',
-    network: {
-      name: 'management network',
-      address: '192.168.0.0/24',
-      isDefault: true,
+if (process.env.NODE_ENV === 'test' || process.env.DATA_SOURCE === 'mock') {
+  MOCK_HOSTS_BY_PROVIDER = [
```

Comment:
Since these aren't indexed by provider anymore let's just rename it to `MOCK_HOSTS`. Nitpick, I know.